.github/workflows/build-releases.yml

```
@@ -0,0 +1,260 @@
+        type: choice
+        options:
+          - beta
+          - stable
+          - nightly
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-matrix:
+    name: Build ${{ matrix.variant }} on ${{ matrix.os }}
```

actionlint (1.7.7)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type {desc: string; features: string; name: string}

(expression)